modules/oxide/src/less/theme/components/advtemplate/advtemplate.less

```
@@ -0,0 +1,25 @@
+//
+// Advtemplate
+//
+
+@advtemplate-iframe-border-style: solid;
+@advtemplate-iframe-border-width: 1px;
+@advtemplate-iframe-border-color: @border-color;
+@advtemplate-iframe-border-radius: @control-border-radius;
```

In the designs for this popup `iframe`’s border radius is larger than regular control radius at 10px, my best guess is that it is done to mimic editor’s border radius (since this preview iframe is basically like a preview of an editor), so I think this might make more sense:

```suggestion
@advtemplate-iframe-border-radius: @tinymce-border-radius;
```